docs/cases/api/actions-api/cases-actions-api-intro.asciidoc

```diff
@@ -15,7 +15,9 @@ which stores the information required to interface with the external system.
 
 {es-sec} uses these external APIs to send cases:
 
-* {sn}: https://developer.servicenow.com/dev.do#!/reference/api/quebec/rest/c_TableAPI[Table API]
+* {sn}: https://developer.servicenow.com/dev.do#!/reference/api/rome/rest/c_ImportSetAPI[Import Set API]
++
+IMPORTANT: {sn} ITSM and SecOps connectors created in {stack} version 7.15.0 or earlier are marked as deprecated after you upgrade to version 7.16.0 or later. It is strongly recommended to <<post-upgrade-deprecated-sn-connector, update deprecated connectors>> or create new ones to ensure you have access to connector enhancements, such as updating incidents. 
```

Comment:
In cases, you can update incidents even with the old API. Only in detections you get this enhancement. Another enhancement is that they can map the data into ServiceNow tables of their choice.

Reply:
Slacked @cnasikas about this and am changing it to say that ITSM and SecOps connectors created in 7.15 or earlier use the V2 Table API and those created in 7.16+ use the Import Set API.